godotenv.go

```diff
@@ -37,10 +37,8 @@ func Load(filenames ...string) (err error) {
 
 	for _, filename := range filenames {
 		err = loadFile(filename)
-		if err != nil {
-			return // return early on a spazout
-		}
 	}
+	// return the last load error
 	return
```

Comment:
Yeah it's this change in behaviour that I've been laying an egg over. I've been reading the orginal ruby library that inspired this (and whose implementation I try to mirror), and I don't think I noticed before that the magic autoloading is part of `dotenv-rails` vs `dotenv` pure. So it seems like the "right" thing to do is closest to the third option, but it's also the most wrong thing to do.